### DISCUSSION TITLE
VUE2  dialog里面打开wangEditor，第一次打开正常，但是后续打开都是v-model的值明明已经改变了，但是还是渲染第一次的

### DISCUSSION THREAD

**Discussion by gushenyi123** [wangeditor-team/wangEditor#4363] [Q&A]
VUE2  做个不同的按钮来渲染同一个dialog；
dialog里面渲染wangEditor；
第一次点击按钮打开渲染数据正常，但是后续（其他按钮）打开v-model渲染的都是第一次打开的数据，明明绑定变量的html值都已经变化了，但是v-mode内容一直不变；
除非我再次去点击第一次那个按钮才能全部正常

**Comment by wangfupeng1988** [comment]
给一个复现的 demo 吧

**Comment by gushenyi123** [comment]
我通过反复的测试和确认发现，如果v-model绑定的元素为空字符串（html: ''）就有这个现象，如果我将初始值改为（`html: '<p></p>'`）就不存在这个问题；
我先暂时就这样用着
